mmd_tools/core/pmx_parser.py

```
@@ -0,0 +1,244 @@
+
+        Raises:
+            IOError: ファイル書き込みに失敗した場合。
+        """
+        try:
+            with open(file_path, "wb") as f:
+                # Header
+                self.header.write(f)
+
+                # Get sizes from header
+                encoding = self.header.encoding
+                vertex_size = self.header.vertex_index_size
```

Ruff (0.12.2)

Local variable `vertex_size` is assigned to but never used

Remove assignment to unused variable `vertex_size`

(F841)